build.gradle

```diff
@@ -88,6 +88,8 @@ dependencies {
 	modRuntimeOnly "objects:client:43db9b498cb67058d2e12d394e6507722e71bb45" // https://piston-data.mojang.com/v1/objects/43db9b498cb67058d2e12d394e6507722e71bb45/client.jar
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
+	include(implementation(annotationProcessor("io.github.llamalad7:mixinextras-fabric:0.5.0-beta.5")))
```

Comment:
Requires a beta version of mixin extras, but should be stable enough to be used. Should be tested for compatibility with other mods.